git.md

```diff
@@ -0,0 +1,50 @@
+# Working with Git
+
+These guidelines applied to **all** CocoaHeads Russia projects.
+
+## General
+
+We use [GitFlow](http://nvie.com/posts/a-successful-git-branching-model/) approach in our projects.
+
+![GitFlow](/resources/images/gitflow.png)
+
+## Branching
+
+**We use [Trello](https://trello.com) for task managements and [this hook](/resporces/scripts/prepare-commit-msg) for git integration with Trello**
```

Comment:
By the way, our board is public
https://trello.com/b/CIz6BNT1/server-side-swift

Reply:
I'll add Trello link to server docs README. This README is about all CH projects.